scripts/dataset_processing/tts/extract_sup_data.py

```diff
@@ -0,0 +1,51 @@
+import torch
+from tqdm import tqdm
+from hydra.utils import instantiate
+from nemo.core.config import hydra_runner
+
+
+def preprocess_ds_for_fastpitch_align(dataloader):
```

Comment:
Is the point of this file just to extract pitch statistics?
Should we run normalizer and save the normalized text to a .json? Instead of running everytime we call train, would save a decent amount of time.